### PR TITLE
Added russian translations

### PR DESCRIPTION
**Pull request by vladislav-puzyrev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ KioskBoard.Merge({
 ### Language (JSON)
 If custom keys are not defined with the `keysArrayOfObjects` option, the `keysJsonUrl` option can be used. Can create an Array of Objects for custom keys related to a custom language. Expecting JSON format is like `[{"key":"value", "key":"value"}, ...]` Each object in that array creates a row element (HTML) on the keyboard. The "key" in the objects is an "index" for each Keyboard Keys. Also, the "value" is each key's value and text.
 
-Additionally, KioskBoard includes 6 different language packages: `English` `Turkish` `Spanish` `German` `French` `Hungarian`
+Additionally, KioskBoard includes 6 different language packages: `English` `Turkish` `Spanish` `German` `French` `Hungarian` `Russian`
```
Comment (furcan) [inline README.md:199]: ...7 different...

File: docs/index.html [hunk 2]
```diff
@@ -362,7 +362,7 @@ <h2 class="sec-title">Merge</h2>
...
...ener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-english.json"><span class="option">English</span></a> <a target="_blank" rel="noopener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-turkish.json"><span class="option">Turkish</span></a> <a target="_blank" rel="noopener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-spanish.json"><span class="option">Spanish</span></a> <a target="_blank" rel="noopener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-german.json"><span class="option">German</span></a> <a target="_blank" rel="noopener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-french.json"><span class="option">French</span></a> <a target="_blank" rel="noopener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-hungarian.json"><span class="option">Hungarian</span></a> <a target="_blank" rel="noopener noreferrer" href="https://github.com/furcan/KioskBoard/blob/master/dist/kioskboard-keys-russian.json"><span class="option">Russian</span></a></p>
```
Comment (furcan) [inline docs/index.html:365]: ...7 different...

**Review by furcan — CHANGES_REQUESTED** [review]
Thank you.

**Comment by vladislav-puzyrev** [comment]
I suggest moving the letter `Ё` to the lower right corner because there is not enough space for it in the top row at the popular resolution of 1080px in width. I think this is not a big problem for the Russian language because on real keyboards the letter `Ё` is sometimes placed differently on the character ~ or / especially since many use the letter `Е` instead of `Ё`.

image

**Review by furcan — APPROVED** [review]

**Pull request merged**